plugins/SlicerT/SlicerT.cpp

```diff
@@ -0,0 +1,666 @@
+/*
+ * SlicerT.cpp - simple slicer plugin
+ *
+ * Copyright (c) 2006-2008 Andreas Brandmaier <andy/at/brandmaier/dot/de>
```

Comment:
Your name here (and wherever else you've copied/pasted the license).

Reply:
done